src/resources/card.rs

```diff
@@ -154,26 +154,26 @@ pub enum CheckResult {
 
 #[derive(Copy, Clone, Debug, Deserialize, Serialize, Eq, PartialEq)]
 pub enum CardBrand {
-    #[serde(rename = "American Express")]
```

Comment:
I don't think this is safe - 
https://stripe.com/docs/api/cards/object#card_object-brand

Would this meet your needs? https://github.com/wyyerd/stripe-rs/pull/188

Reply:
#188 has now been merged. The `enum PaymentMethodCardBrand` now holds variants that would be helpful here.

Reply:
Yes, #188 covers.  Thanks!